### ISSUE TITLE
Fix charts docs a11y violations

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#7675]
-  Duplicate IDs (probably shared between examples)
   -  Stack chart
   -  Tooltip chart
   -  Scatter chart
   -  Line chart
   -  Donut utilization chart
   -  Bullet chart
   -  Area chart
   -  Legend chart
   -  Bar chart
   -  Pie chart
   -  Donut chart
-  SVGs need alt text
   -  Tooltip chart
   -  Sparkline chart
   -  Bullet chart
-  Legend chart examples - interactive controls cannot be nested
